SharedPackages/BrowserServicesKit/Sources/PrivacyDashboard/BrokenSiteReporting/PerformanceMetricsSubfeature.swift

```diff
@@ -35,9 +36,23 @@ public class PerformanceMetricsSubfeature: Subfeature {
     }
 
     public func handler(forMethodNamed methodName: String) -> Handler? {
-        guard methodName == "vitalsResult" else { return nil }
+        switch methodName {
+        case "vitalsResult":
+            return vitalsResult
```

Comment:
I think we can delete this now and the respective method that handles it. The getVitals can go too right, we've moving that to the breakageReporting subfeature.